app.py

```diff
@@ -1,79 +1,114 @@
 from flask import Flask, render_template, request, redirect, url_for
 import session_items as session
-from item import TrelloItem
+from item import Item
 from trello import TrelloClient
 from viewModel import ViewModel
 import os
+import pymongo
+from datetime import datetime
+from bson.objectid import ObjectId
+
+date_time_format = "%Y-%m-%dT%H:%M:%S.%fZ"
+
 
 def get_trello_board(api_key, token, board_id):
```

Comment:
We should remove all of the trello legacy code now, and as part of that we should remove the py-trello package as well